src/scss/3_components/form/form.scss

```diff
@@ -3,106 +3,275 @@
  * Provides the generic styling for all types of form components.
  */
 @import '../../init';
+@import 'button/button-defs';
 
-fieldset,
-.fieldset {
-  border: 0;
+main form {
+  margin: 0;
+  padding: 0;
+  max-width: 800px;
 
-  &:focus {
-    outline: 0;
+  .form-actions {
+    margin: 2.5em 0;
+  }
+
+  .form-item {
+    position: relative;
+  }
+
+  & > .form-item,
+  .form-wrapper > .form-item {
+    padding: 0;
+    margin: 0 0 2em;
+  }
+
+  .description {
+    @include rhythm-margins(1, 1.66667);
+    @include adjust-font-size-to(13px, 2.6667);
+    @include breakpoint($phab, $no-query: true) {
+      @include adjust-font-size-to(15px, 3);
+    }
+  }
+
+  fieldset,
+  .fieldset {
     border: 0;
+    padding: 0;
+
+    &:first-child {
+      margin-top: 0;
+    }
+
+    &:focus {
+      outline-color: $brand-primary-medium;
+      border: 0;
+    }
+
+    & + div.form-item {
+      margin-top: -.5em !important;
+    }
+
+    & > .form-item {
+      margin: 1em 0 0 0;
+    }
   }
-}
 
-select,
-[type='text'],
-[type='tel'],
-[type='url'],
-[type='email'],
-[type='password'],
-[type='date'],
-[type='time'],
-[type='number'],
-textarea {
-  box-sizing: border-box;
-  border: 1px solid #333;
-  border-radius: 0;
-  color: #333;
-  margin: .6rem 0 0;
-  padding: .6rem;
-
-  &:focus {
-    border-color: #2daff7;
-    box-shadow: 0 0 3px #2daff7;
-    color: $black;
-    outline: 0;
-  }
-
-  /* Highlight the form elements that caused a form submission error. */
-  &.error,
-  .error:focus {
-    border: 1px solid #c94b4b;
-    background-color: rgba(255, 182, 193, .5); // lightpink
-    box-shadow: none;
-    color: #000;
+  legend,
+  fieldset h2,
+  .fieldset h2 {
+    @include responsive-typo-map($form-legend);
+    color: $brand-primary-dark;
+    font-weight: normal;
+  }
+
+  label, span.label {
+
+    @include responsive-typo-map($body-text);
+
+    font-weight: normal;
+
+    & + .form-radios {
+      margin-top: 3px;
+      margin-top: .3rem;
+      @include breakpoint($phab, $no-query: true) {
+        margin-top: 6px;
+        margin-top: .6rem;
+      }
+    }
+
+    &.element-invisible + .form-radios {
```

Comment:
the `.element-invisible` class should no longer work on D8 and has been replaced in core with `.visually-hidden`, therefore we should try & align our code with this as well

Reply:
Yep - will do